bookwyrm/models/__init__.py

```diff
@@ -26,6 +26,7 @@
 from .group import Group, GroupMember, GroupMemberInvitation
 
 from .import_job import ImportJob, ImportItem
+from .bookwyrm_import_job import BookwyrmImportJob
```

Comment:
From memory this was because we have `from bookwyrm import models` in `BookwyrmExportJob` and we have up trying to resolve a horrendous loop of infinitely-self-referencing references. If you can see a way to get around this I agree it would be better.

Reply:
I think this will do it: https://github.com/bookwyrm-social/bookwyrm/pull/3085